beprof/profile.py

```diff
@@ -144,11 +146,14 @@ def normalize(self, dt):
         except RuntimeWarning as e:
             logger.error('in normalize(). self class is {0}, dt={1}'.format(self.__class__, dt))
             raise Exception("Scaling factor error:\n" + str(e))
-        self.y /= ave
+        self.y = self.y / ave
```

Comment:
Sure, the second came to my mind when I was writing this reply, so...

What about the `div zero` case? Should we somehow catch it, or just let it error or whatever?